addons/stock/i18n/stock.pot

```diff
@@ -6,8 +6,8 @@ msgid ""
 msgstr ""
 "Project-Id-Version: Odoo Server 17.0\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2023-10-26 21:54+0000\n"
```

Comment:
No need to export/import the .pot file for a small change. Just remove the sentences you deleted and add the new ones at the end